hw/ip/spi_device/doc/_index.md

```diff
@@ -677,15 +677,25 @@ If `invalid_locality` configuration is set, the logic returns `INVALID` value to
 If the request is in the supported locality (0-4), the logic checks `TPM_ACCESS_x.activeLocality` then returns data based on the table 39 in the spec for Return-by-HW registers.
 Other registers in the table should be processed by SW.
 
+## Countermeasures
+
+This version of the SPI_DEVICE IP implements the parity to detect bit flip errors on the internal SRAM.
+The HW checks the parity error when the SW reads data from the SRAM.
+The error is reported to the SW via TL D channel error signal.
+SW is recommended to discard the current context if any transaction is ongoing then to reset the IP.
+
 # Design Details
 
 ## Clock and Phase
 
-The SPI device module has two programmable register bits to control the SPI
-clock, {{< regref "CFG.CPOL" >}} and {{< regref "CFG.CPHA" >}}. CPOL controls clock polarity and CPHA controls the clock
-phase. For further details, please refer to this diagram from Wikipedia:
+The SPI device module has two programmable register bits to control the SPI clock, {{< regref "CFG.CPOL" >}} and {{< regref "CFG.CPHA" >}}.
+CPOL controls clock polarity and CPHA controls the clock phase.
+For further details, please refer to this diagram from Wikipedia:
 [File:SPI_timing_diagram2.svg](https://en.wikipedia.org/wiki/Serial_Peripheral_Interface#/media/File:SPI_timing_diagram2.svg)
 
+This version of SPI_DEVICE HWIP supports mode 0 (CPHA and CPOL as 0) and mode 3 (CPHA and CPOL as 1) for Generic, Flash, and Passthrough modes.
```

Comment:
we can support mode1 and mode2 for the Generic mode. Do you want to mention that?

Reply:
Even the module runs without any errors on mode 1 or 2, my intention of the design is to support mode 0 and 3 only (also in Generic mode). So I'd rather stick to this.

Reply:
SGTM